### PR TITLE
[MRG][DOC] Tweaks to the docs

### PR DESCRIPTION
**Pull request by jasmainak**
cc @chenghuzi @rythorpe

### CODE REVIEW

**Comment by jasmainak** [comment]
Tests pass now!

**Comment by jasmainak** [comment]
https://output.circle-artifacts.com/output/job/deb24eb7-40b2-47cd-bd2b-1b249df3868d/artifacts/0/html/gui/index.html

File: doc/gui/index.rst [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 
 The brand-new lightweight web-based hnn GUI that works with hnn-core.
 
+.. raw:: html
 
-Launch GUI in a Jupyter notebook
-================================
+    <div class="sphx-glr-thumbnails">
+        <div class="sphx-glr-thumbcontainer"
+            tooltip="Launch GUI in a Jupyter notebook">
```
Comment (rythorpe) [inline doc/gui/index.rst:11]: Why is the tooltip text different from the title? Something about it isn't intuitive to me.

File: doc/gui/index.rst [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
+     :alt: 01. Basic GUI usage
 
-See :doc:`tutorial_erp`
+  :doc:`basic_gui_usage`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">01. Basic GUI usage</div>
+    </div>
+    <div class="sphx-glr-thumbcontainer" tooltip="GUI ERP example">
```
Comment (rythorpe) [inline doc/gui/index.rst:24]: Same as above.

**Comment by jasmainak** [comment]
made the tooltip consistent!

File: doc/gui/index.rst [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
+     :alt: 01. Basic GUI usage
 
-See :doc:`tutorial_erp`
+  :doc:`basic_gui_usage`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">01. Basic GUI usage</div>
+    </div>
+    <div class="sphx-glr-thumbcontainer" tooltip="Basic GUI usage">
```
Comment (rythorpe) [inline doc/gui/index.rst:24]: Something weird is going on here but I don't know enough about rST + Sphinx Gallery to troubleshoot: [the "Basic GUI usage" tooltip is being applied to both thumbnails](https://output.circle-artifacts.com/output/job/4b3e839d-5c0d-4a39-9162-0a8814bcafb9/artifacts/0/html/gui/index.html).

Comment (rythorpe) [reply doc/gui/index.rst:24]: As soon as this is fixed I'll merge @jasmainak!

**Comment by jasmainak** [comment]
fixed tooltip!

**Comment by rythorpe** [comment]
Thanks @jasmainak!

**Pull request merged**